docs/operations/cx-server-operations-guide.md

```diff
@@ -72,31 +79,38 @@ The status command provides basic overview about your Cx Server instance.
 ``` 
 
 ##### stop
+
 The Cx Server can be stopped with the `stop` command.
+
 ```bash
 ./cx-server stop
 ``` 
-This stops the Jenkins Docker container if it is running. A subsequent `start` command restores the container.
+This stops the Jenkins Docker container if it is running.
+A subsequent `start` command restores the container.
```

Comment:
Does not related to the topic of the PR (Document release process)  - like also all the newlines. Blurs the focus of the PR and makes it harder to read.